frontend/providers/applaunchpad/public/locales/en/translation.json

```diff
@@ -0,0 +1,123 @@
+{
+  "Applications": "Applications",
+  "New Application": "New Application",
+  "Cancel": "Cancel",
+  "Confirm": "Confirm",
+  "Name": "Name",
+  "Status": "Status",
+  "Creation Time": "Creation Time",
+  "CPU": "CPU",
+  "Memory": "Memory",
+  "Number of Instances": "Number of Instances",
+  "Storage Capacity": "Storage Capacity",
+  "Operation": "Operation",
+  "Running": "Running",
+  "Paused": "Paused",
+  "Active": "Active",
+  "Details": "Details",
+  "Start Up": "Start Up",
+  "Delete": "Delete",
+  "Pause": "Pause",
+  "Restart": "Restart",
+  "Change": "Change",
+  "ApplicationType": "ApplicationType",
+  "Basic Information": "Basic Information",
+  "Deployment Mode": "Deployment Mode",
+  "Advanced Configuration": "Advanced Configuration",
+  "Start Command": "Start Command",
+  "Run Parameters": "Run Parameters",
+  "Not Configured": "Not Configured",
+  "Accessible to the Public": "Accessible to the Public",
+  "Configuration File": "Configuration File",
+  "Storage Volume": "Storage Volume",
+  "Environment Variables": "Environment Variables",
+  "Fixed Number of Instances": "Fixed Number of Instances",
+  "Image Name": "Image Name",
+  "Private": "Private",
+  "Public": "Public",
+  "Image Name (Private)": "Image Name (Private)",
+  "Real-time Monitoring": "Real-time Monitoring",
+  "Network Configuration": "Network Configuration",
+  "Intranet Address": "Intranet Address",
+  "Internet Address": "Internet Address",
+  "Not Enabled": "Not Enabled",
+  "Update Time": "Update Time",
+  "Continue": "Continue",
+  "Terminal": "Terminal",
+  "Log": "Log",
+  "None": "None",
+  "Intelligent Analysis": "Intelligent Analysis",
+  "Problem Analysis": "Problem Analysis",
+  "Export": "Export",
+  "Edit Environment Variables": "Edit Environment Variables",
+  "Application Deployment": "Application Deployment",
+  "Creating": "Creating",
+  "Abnormality Detected": "Abnormality Detected",
+  "Change Application": "Change Application",
+  "Deploy Application": "Deploy Application",
+  "Confirm Deploy Application?": "Confirm Deploy Application?",
+  "Deployment Successful": "Deployment Successful",
+  "Deployment Failed": "Deployment Failed",
+  "Update Application": "Update Application",
+  "Confirm Update Application?": "Confirm Update Application?",
+  "Update Successful": "Update Successful",
+  "Update Failed": "Update Failed",
+  "Prompt": "Prompt",
+  "jump_prompt": "Jump prompt",
+  "jump_message": "This application does not allow standalone usage. Click OK to proceed to Sealos Desktop.",
```

Comment:
This application cannot be used independently. Please click 'OK' to proceed to Sealos Desktop.